src/Fmt_ast.ml

```diff
@@ -2311,7 +2341,7 @@ and fmt_class_type c ?(box = true) ({ast= typ} as xtyp) =
       in
       hvbox_if box 0 (list xt1N "@;-> " fmt_arg)
   | Pcty_extension ext -> fmt_extension c ctx "%" ext
-  | Pcty_open (flag, lid, cl) ->
+  | Pcty_open ({popen_override= flag; popen_expr= lid; _}, cl) ->
```

Comment:
handle `popen_loc` and `popen_attributes` ..

Reply:
Fixed by reusing `fmt_open_description`. Didn't find example for attributes but fix dropped comments.

Reply:
```
class type a = let open [@aaaaaaaaaaaaa] A in b
```

Reply:
Thanks. This was not handled correctly so I fixed it.